.azure-pipelines/azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: Maven@3
+  inputs:
+    mavenPomFile: 'pom.xml'
+    mavenOptions: '-Xmx3072m'
+    javaHomeOption: 'JDKVersion'
+    jdkVersionOption: '1.8'
+    jdkArchitectureOption: 'x64'
+    publishJUnitResults: true
```

Comment:
whats this 😄

Reply:
see the `Publish to Azure Pipelines` item in https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/maven?view=azure-devops